Clang 15.0.3 fails build because of  missing prototype in Commix.

Clang 15.0.3 started reported an error building the indicated file.

```
/scala-native/gc/commix/Heap.c:239:33: warning: passing arguments to 'WeakRefGreyList_CallHandlers' without a prototype is deprecated in all versions of C and is not supported in C2x [-Wdeprecated-non-prototype]
[error]     WeakRefGreyList_CallHandlers(heap);
[error]                                 ^
```

Is there anyone looking out for Commix?   If not, should I try to provide the indicated function prototype
 (or, if it already exists), why it is not being used/seen.

Please advise.

I did not see this error with Clang 15.0.2.  For now, I will fall back to an earlier version of Clang.

--- a/nativelib/src/main/resources/scala-native/gc/commix/Heap.c
+++ b/nativelib/src/main/resources/scala-native/gc/commix/Heap.c
@@ -236,7 +236,7 @@ void Heap_Collect(Heap *heap) {
                       heap->mark.currentEnd_ns);
     Phase_Nullify(heap, stats);
     Phase_StartSweep(heap);
-    WeakRefGreyList_CallHandlers(heap);
+    WeakRefGreyList_CallHandlers();
 }
 
 bool Heap_shouldGrow(Heap *heap) {

